sdk/monitor/azure-monitor-query/samples/sample_batch_query.py

```diff
@@ -31,8 +31,9 @@
         workspace= os.environ['LOG_WORKSPACE_ID']
     ),
     LogsQueryRequest(
-        query= "AppRequests | take 2",
-        workspace= os.environ['LOG_WORKSPACE_ID']
+        query= "AppRequests",
+        workspace= os.environ['LOG_WORKSPACE_ID'],
+        include_statistics=True
```

Comment:
should sample also have `include_render`, `server_timeout` or nah?

Reply:
we have a different sample for server time out - for render we are avoiding for now